### PR TITLE
Validate json schema + sets

### PR DESCRIPTION
**Pull request by sierra-moxon**

### CODE REVIEW

File: json_schema_example_data/disease_annotation_example.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "disease_agm_annotation_set": [
+    {
+      "created_by": "Test User String",
```
Comment (sierra-moxon) [inline json_schema_example_data/disease_annotation_example.json:4]: Person is the range here

**Comment by oblodgett** [comment]
I'm trying to figure out when the artifacts are checked in at the root of the repo? 

If also appears that this pull request changes how the build is done.

There are no instructions on how to build this repo other then trying to read the Makefile and .github/workflow/* task

It also appears that this new build just puts everything into the root of the repo as in found java stuff in this directory:
org/alliancegenome/curation/model contains a bunch of java classes

python seems to be generated into the python/* directory.

I see a jsonschema directory, in the root but it doesn't have the schema files in however maybe everything is under the allianceModel.schema.json file?

I see this entry for the Ingest:

 ```json
       "Ingest": {
         "additionalProperties": false,
         "description": "",
         "properties": {
            "allele_ingest_set": {
               "items": {
                  "$ref": "#/$defs/Allele"
               },
               "type": "array"
            },
            "disease_annotation_ingest_set": {
               "items": {
                  "$ref": "#/$defs/DiseaseAnnotation"
               },
               "type": "array"
            }
         },
         "required": [],
         "title": "Ingest",
         "type": "object"
      },
 ```

It would appear that Allele's get submitted with disease annotations, which I would guess is not the intent?

File: src/schema/allianceModel.yaml [hunk 2]
```diff
@@ -44,3 +44,37 @@ subsets:
   alliance subset:
     description: >-
       Subset consisting of just the alliance activities
+
+slots:
+  allele ingest set:
```
[the remark this answers is not in the document]
Comment (sierra-moxon) [reply src/schema/allianceModel.yaml]: yep!  fixed - lots of debugging here between what linkml expects, our conventions and the sets workaround.  This was one of my iterations to confirm which piece of the code was causing my error.

File: src/schema/ingest.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+id: https://github.com/alliance-genome/agr_curation_schema/src/schema/ingest
+name: Alliance-Schema-Ingest
+title: Alliance Schema Ingest
+
```
[the remark this answers is not in the document]
Comment (sierra-moxon) [reply src/schema/ingest.yaml]: lots of debugging here between what linkml expects, our conventions, and the sets workaround.  This was one of my iterations to confirm which piece of the code was causing my error.  all of the ingest bits are moved now to the ingest.yaml file specifically.

**Comment by sierra-moxon** [comment]
> It would appear that Allele's get submitted with disease annotations, which I would guess is not the intent?

No, the Ingest object is just a placeholder object to allow us to make slots appear in the top-level JSON schema (that we can then use in the validator).  A quirk, but also a handy workaround.

**Pull request merged**